src/pages/index.js

```diff
@@ -14,81 +16,190 @@ const addCardAddButton = document.querySelector(selectors.cardAddButton);
 const updateProfileButton = document.querySelector(selectors.profileEditButton);
 const cardFormElement = document.querySelector(selectors.cardPopup);
 const profileFormElement = document.querySelector(selectors.profilePopup);
+const avatarFormElement = document.querySelector(selectors.avatarPopup);
 const profileHeadingInput = profileFormElement.querySelector(selectors.profileName);
 const profileDescriptionInput = profileFormElement.querySelector(selectors.profileDescription);
 const newImagePopup = new PopupWithImage(selectors.cardImagePopup);
+const updateAvatarButton = document.querySelector(selectors.avatarEditButton);
+const avatarURLInput = profileFormElement.querySelector(selectors.avatarURL);
+
+// Initialize popups and form validators
 newImagePopup.setEventListeners();
 const editProfileFormValidator = new FormValidator(formValidationConfig, profileFormElement);
 const cardFormValidator = new FormValidator(formValidationConfig, cardFormElement);
-editProfileFormValidator.enableValidation();
-cardFormValidator.enableValidation();
-
-//INITIAL CARDS
-const cardSection = new Section(
-  { items: initialCards,
-    renderer: (item) => {
-      const card = renderCard(item);
-      cardSection.addItem(card);
-    }
+const updateAvatarFormValidator = new FormValidator(formValidationConfig, avatarFormElement);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```